GNUmakefile

```diff
@@ -11,7 +25,16 @@ test: fmtcheck
 	echo $(TEST) | \
 		xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
 
-testacc: fmtcheck
+testacc:
+	@:$(call check_defined, \
+						ARM_CLIENT_ID \
```

Comment:
Environment VAR is not the only way to authorize terraform to provision resource.

Reply:
@metacpp this is to run the Acceptance Tests - which are only supported by Service Principals